video2dataset/audio_utils.py

```diff
@@ -0,0 +1,76 @@
+import json
```

Comment:
let's not have an utils file, and move this into downloader

Reply:
ok